docs: uitbreiden informatie en praktische tips voor sprekers heartbeat (#2325) (#2326)
nl-design-system/documentatie: docs/community/events/heartbeat/zelf-iets-vertellen.mdx

---
title: Zelf iets vertellen · Heartbeat · Bijeenkomsten
hide_title: true
hide_table_of_contents: false
sidebar_label: Zelf iets vertellen
pagination_label: Wat moet ik weten als ik zelf iets wil vertellen tijdens de Heartbeat?
description: Wat moet ik weten als ik zelf iets wil vertellen tijdens de Heartbeat?
slug: /events/heartbeat/zelf-iets-vertellen
---

# Zelf iets vertellen?

> In de Heartbeat vertelt het kernteam van NL Design System elke twee weken wat de laatste stand van zaken is. Ook geven we het woord aan mensen uit de community. Werk jij met of aan het NL Design System? Dan kun je zelf ook iets presenteren!

Reviewer: Elke twee weken deelt het kernteam van het NL Design System de laatste ontwikkelingen in de Heartbeat.

> We moedigen bijdragen vanuit de community aan. Wil je iets vertellen tijdens een Heartbeat? Lees dan hieronder hoe dat werkt.

Reviewer: We moedigen bijdragen vanuit de community van harte aan. Wil je iets vertellen tijdens een Heartbeat? Lees hieronder hoe dat werkt.

> We moedigen bijdragen vanuit de community aan. Wil je iets vertellen tijdens een Heartbeat? Lees dan hieronder hoe dat werkt.

Reviewer: ```suggestion
We moedigen bijdragen vanuit de community van harte aan. Wil je iets vertellen tijdens een Heartbeat? Lees hieronder hoe dat werkt.
```

## Welke onderwerpen passen bij de Heartbeat?

> Je presentatie moet relevant zijn voor NL Design System. Bijvoorbeeld:

Reviewer: ```suggestion
Je presentatie moet relevant zijn voor het NL Design System. Denk bijvoorbeeld aan:
```

- Hoe jouw organisatie NL Design System gebruikt.
- Nieuwe componenten die je hebt ontwikkeld.
- Verbeteringen aan bestaande componenten.
- Best practices en geleerde lessen.

> - Samenwerking van teams rondom NL Design System.

Reviewer: ```suggestion
- Samenwerking tussen teams rondom NL Design System.
```

- Heb je een ander goed idee? Laat het ons weten!

> ## Hoe werkt het aanmelden?

Reviewer: ```suggestion
## Hoe meld je je aan?
```

Neem contact op met het kernteam in het #nl-design-system-developers kanaal in de [Slack groep](http://praatmee.codefor.nl/) van Code for NL of stuur een e-mail naar [info@nldesignsystem.nl](mailto:info@nldesignsystem.nl).

Wat willen we weten?

- Waar wil je over vertellen?

> - Hoeveel tijd heb je nodig?

Reviewer: ```suggestion
- Hoeveel tijd denk je nodig te hebben?
```

> Om je presentatie te introduceren, willen we ook graag weten:

Reviewer: ```suggestion
Om je goed te kunnen introduceren, horen we ook graag:
```

- Wat is de titel van je presentatie?

> - Hoe wil je geintroduceerd worden? (Bijvoorbeeld: je naam, functie, organisatie, en een korte beschrijving van je presentatie.)

Reviewer: ```suggestion
- Hoe wil je worden geïntroduceerd? (Bijvoorbeeld: je naam, functie, organisatie en een korte beschrijving van je presentatie.)
```

> We nemen dan contact met je op om de sessie in te plannen. Als het goed is krijg je daarna een uitnodiging in je agenda.

Reviewer: "Als het goed is krijg je daarna een uitnodiging in je agenda."

Sturen wij een Outlook uitnodiging door? Hoe werkt dit precies? Dit is me nog niet helemaal duidelijk.

Author: @Robbert dit had ik uit de email die jij me stuurde. Heb jij meer context?

Reviewer 2: @eslook @Rozerinay Ik stuurde zelf meestal een uitnodiging naar iedereen die presenteerde, bijvoorbeeld "🎤 Presentatie geven in Heartbeat", zodat we het risico van een no-show verkleinen.

Reviewer: @eslook yes lijkt me goed! Of iets korter:

We nemen daarna contact met je op om de sessie in te plannen. Je ontvangt vervolgens een uitnodiging in je agenda.

Author: ```suggestion
We nemen daarna contact met je op om de sessie in te plannen. Je ontvangt vervolgens een uitnodiging in je agenda.
```

## Hoeveel tijd krijg ik?

> De Heartbeat duurt 1 uur. Geef van te voren aan hoe lang je presentatie duurt, zodat we de tijd goed kunnen verdelen en rekening kunnen houden met andere sprekers.

Reviewer 2: ```suggestion
De Heartbeat maximaal 45 minuten. Geef van te voren aan hoe lang je presentatie duurt, zodat we de tijd goed kunnen verdelen en rekening kunnen houden met andere sprekers.
```

Reviewer: ```suggestion
De Heartbeat duurt maximaal 45 minuten. Laat vooraf weten hoe lang je presentatie duurt, zodat we de tijd goed kunnen indelen met andere sprekers.

```

> Er is geen minimale tijd voor een presentatie. De meeste presentaties zijn ongeveer 20-25 minuten, met daarna 5-10 minuten voor vragen. Iets uitlopen is geen probleem. Je kan ook een kortere presentatie geven, bijvoorbeeld 10 minuten.

Reviewer: ```suggestion
Er is geen minimale tijd: de meeste presentaties duren 20–25 minuten, met 5–10 minuten voor vragen. Iets korter of langer mag ook.```

## Waar moet ik rekening mee houden?

> - Kom graag 15 minuten van tevoren online. Dat helpt ons om nog een korte technische check te doen en te zorgen dat alles werkt.

Reviewer 2: ```suggestion
- Kom graag 10 minuten van tevoren online. Dat helpt ons om nog een korte technische check te doen en te zorgen dat alles werkt.
```

Reviewer: ```suggestion
- Kom 10 minuten van tevoren online voor een korte technische check.
```

- De sessie vindt plaats via Microsoft Teams. De link staat in je kalenderuitnodiging.

> - We nemen de presentatie op en publiceren deze na afloop op de publieke site, tenzij je bezwaar maakt of we anders hebben afgesproken.

Reviewer: ```suggestion
- De presentatie wordt opgenomen en na afloop gepubliceerd op de publieke site, tenzij je hier bezwaar tegen maakt of we iets anders hebben afgesproken.
```

> - Kun je je slides van tevoren aanleveren? We kijken er niet inhoudelijk naar (tenzij je feedback wilt, die geven we graag), maar geven ze aan de schrijftolk zodat die zich kan voorbereiden.

Reviewer: ```suggestion
- Kun je je slides van tevoren aanleveren? We kijken er inhoudelijk niet naar (tenzij je om feedback vraagt, die geven we graag), maar sturen ze wel door naar de schrijftolk zodat die zich kan voorbereiden.
```

## Tips voor presenteren

> - Vertel ook in woorden wat je op je slides laat zien. Zo blijft de presentatie ook zonder beeld duidelijk.

Reviewer: ```suggestion
- Benoem wat er op je slides staat, zodat de presentatie ook zonder beeld te volgen is.
```

> - Spreek duidelijk en niet te snel.

Reviewer 2: ```suggestion
- Spreek niet te snel, en zo duidelijk mogelijk.
```

- Deel je scherm? Let erop dat je geen persoonlijke informatie laat zien (zoals bladwijzers of formulieren welke automatisch worden ingevuld). Je kunt een gastvenster (of gastprofiel) van je browser gebruiken.

> - Let erop dat je scherm goed leesbaar is voor iedereen. Heb je een heel groot scherm? Deel dan een deel van je scherm of verlaag je resolutie.

Reviewer: ```suggestion
- Zorg dat je scherm goed leesbaar is voor alle kijkers. Heb je een groot scherm? Deel dan een deel ervan of verlaag de resolutie.
```

> - Wil je meer weten over toegankelijk presenteren? Bekijk dan [Preparing slides and projected material](https://www.w3.org/WAI/teach-advocate/accessible-presentations/#speakers-planning-your-session%23preparing-slides-and-projected-material) (Engels).

Reviewer: ```suggestion
- Meer weten over toegankelijk presenteren? Bekijk [Preparing slides and projected material](https://www.w3.org/WAI/teach-advocate/accessible-presentations/#speakers-planning-your-session%23preparing-slides-and-projected-material) (Engelstalig).
```

## Help mee met promotie!

> Je kunt het event helpen promoten. Deel bijvoorbeeld de link naar de website, vertel je collega's dat je gaat spreken, of post op LinkedIn. Je kunt ook als spreker in het LinkedIn-event getoond worden, daar ontvang je een uitnodiging voor.

Reviewer: ```suggestion
Wil je helpen met de promotie van het event? Deel de link naar de website, vertel collega’s dat je gaat spreken of post iets op LinkedIn. Je kunt ook als spreker zichtbaar zijn in het LinkedIn-event. Daar krijg je een uitnodiging voor.
```

## Gedragscode

Tijdens de Heartbeat geldt de [NL Design System Code of Conduct](https://github.com/nl-design-system/.github/blob/main/CODE_OF_CONDUCT.nl.md). Door je aan te melden of te spreken, ga je hiermee akkoord.

> 

Reviewer 2: ```suggestion

<!-- Met "ongepast" verwijzen we ook naar de Code of Conduct en de handhaving. -->

Het is ongepast om je Heartbeat-presentatie te gebruiken om expliciet reclame te maken voor commerciële doeleinden. Je bedrijfsnaam en logo in beeld hebben is geen probleem, een sales-pitch is niet de bedoeling. Overleg vooraf met het kernteam als je twijfelt over je presentatie.
```

We nemen daarna contact met je op om de sessie in te plannen. Je ontvangt vervolgens een uitnodiging in je agenda.

## Hoeveel tijd krijg ik?

De Heartbeat duurt maximaal 45 minuten. Laat vooraf weten hoe lang je presentatie duurt, zodat we de tijd goed kunnen indelen met andere sprekers.

Er is geen minimale tijd: de meeste presentaties duren 20–25 minuten, met 5–10 minuten voor vragen. Iets korter of langer mag ook.

## Waar moet ik rekening mee houden?

- Kom 10 minuten van tevoren online voor een korte technische check.
- De sessie vindt plaats via Microsoft Teams. De link staat in je kalenderuitnodiging.
- De presentatie wordt opgenomen en na afloop gepubliceerd op de publieke site, tenzij je hier bezwaar tegen maakt of we iets anders hebben afgesproken.
- Kun je je slides van tevoren aanleveren? We kijken er inhoudelijk niet naar (tenzij je om feedback vraagt, die geven we graag), maar sturen ze wel door naar de schrijftolk zodat die zich kan voorbereiden.

## Tips voor presenteren

- Benoem wat er op je slides staat, zodat de presentatie ook zonder beeld te volgen is.
- Spreek niet te snel, en zo duidelijk mogelijk.

> - Deel je scherm? Let erop dat je geen persoonlijke informatie laat zien (zoals bladwijzers of formulieren welke automatisch worden ingevuld). Je kunt een gastvenster (of gastprofiel) van je browser gebruiken.

Reviewer: ```suggestion
- Deel je je scherm? Let erop dat je geen persoonlijke informatie laat zien (zoals bladwijzers of formulieren welke automatisch worden ingevuld). Je kunt een gastvenster (of gastprofiel) van je browser gebruiken.
```
